checker/examples/Makefile

```diff
@@ -4,7 +4,6 @@ all: compile
 
 compile:
 	$(JAVAC) *.java
-	$(JAVAC) units-extension/*.java
```

Comment:
(These comments are NOT comments on the code as it appears now.  These comments were on a commit that was later rebased to include more changes.)